packages/vaadin-grid/test/keyboard-navigation.test.js

```diff
@@ -1827,6 +1827,34 @@ describe('keyboard navigation', () => {
 
       expect(spy.callCount).to.equal(1);
     });
+
+    it('should dispatch cell-focus on keyboard navigation', (done) => {
+      tabToBody();
+      right();
+
+      const spy = sinon.spy();
+      grid.addEventListener('cell-focus', spy);
+
+      grid.addEventListener('cell-focus', (e) => {
```

Comment:
You don't need to add another event listener. The event can be obtained from the spy also (first argument of the first call).

Reply:
I tried to rewrite the test like this:
```
const spy = sinon.spy();
grid.addEventListener('cell-focus', spy);

left();

const e = spy.firstCall.args[0];
const context = e.target.getEventContext(e);

expect(context).to.deep.equal(expectedContext);
```
but the `e` obtained from the spy doesn't return any `composedPath` for some reason.

![e-no-composed-path](https://user-images.githubusercontent.com/3392815/112469447-2b77ed00-8d72-11eb-84c6-70bc36812449.png)

Reply:
Ah, I see. Then the callback is needed.

Reply:
Ok I pushed a refactor with some changes